src/components/ordersAddPage.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <h2 class="text-center">Add Order</h2>
```

Comment:
What's this whole page for?

Reply:
This is the addOrder component I started, I believe you continued working on it and perhaps saved it as something else?